### PR TITLE
chore(template): update npm build script to copy adaptiveCards folder

### PR DESCRIPTION
**Pull request by dooriya**
Fix [Bug 14369953](https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/14369953): [GA Bug Bash] Bot App tsconfig.ts not aligned with compilation behavior.

E2E TEST: covered by UI test

### CODE REVIEW

File: templates/bot/ts/command-and-response/package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "scripts": {
         "dev:teamsfx": "env-cmd --silent -f .env.teamsfx.local npm run dev",
         "dev": "nodemon --watch ./src --exec node --inspect=9239 --signal SIGINT -r ts-node/register ./src/index.ts",
-        "build": "tsc --build",
+        "build": "tsc --build && shx cp -r ./src/adaptiveCards ./lib/src",
```
Comment (swatDong) [inline templates/bot/ts/command-and-response/package.json:11]: Does this conflict with #4962 ?

**Review by swatDong — APPROVED** [review]

**Review by IvanJobs — APPROVED** [review]
lgtm

File: templates/bot/ts/command-and-response/package.json (hunk 1 above)
Comment (dooriya) [reply templates/bot/ts/command-and-response/package.json:11]: Ah~yeah, then @KennethBWSong will need to merge with this change :)

Comment (KennethBWSong) [reply templates/bot/ts/command-and-response/package.json:11]: Already include in pr #4962 , thx!

**Pull request merged**